### ISSUE TITLE
Resize with Pillow to avoid downsampling artifacts

### ISSUE THREAD

**Issue by Dipet** [albumentations-team/albumentations#1024]
OpenCV doesn't use antialiasing when downsampling images.
More info:
- [arxiv](https://opendatascience.slack.com/archives/C2ME5EVK7/p1631195105141700)
- [twitter discussion](https://twitter.com/jaakkolehtinen/status/1258102168176951299)
- [ODS discussion](https://opendatascience.slack.com/archives/C2ME5EVK7/p1631195105141700)

**Comment by zakajd** [comment]
There is a separate GitHub repo dedicated to this problem (and working solution!):
https://github.com/assafshocher/ResizeRight

**Comment by fabiofumarola** [comment]
Yes, but still the results are different between PILLOW, OpenCV and ResizeRight. If it ok I'll work on an update to let the resize using PILLOW or OpenCV based on a parameter

**Comment by ternaus** [comment]
I do not see right now, how to add resize with Pillow or pyright in a smooth way. Open to proposals.
